modules/fedimint-lnv2-tests/src/bin/self_payment.rs

```diff
@@ -90,6 +92,64 @@ async fn main() -> anyhow::Result<()> {
             serde_json::to_value(FinalReceiveState::Claimed).expect("JSON serialization failed"),
         );
 
+        // test self payment with two gateways
+
+        let gw_cln = dev_fed.gw_cln().await?;
+
+        fed.pegin_gateway(100_0000, gw_cln).await?;
+
+        let (invoice, receive_op) = serde_json::from_value::<(Bolt11Invoice, OperationId)>(
+            cmd!(
+                client,
+                "module",
+                "lnv2",
+                "receive",
+                gw_cln.addr.clone(),
+                "1000"
+            )
+            .out_json()
+            .await?,
+        )?;
+
+        let send_op = serde_json::from_value::<OperationId>(
+            cmd!(
+                client,
+                "module",
+                "lnv2",
+                "send",
+                gw_lnd.addr.clone(),
+                invoice.to_string()
+            )
+            .out_json()
+            .await?,
+        )?;
+
+        assert_eq!(
+            cmd!(
+                client,
+                "module",
+                "lnv2",
+                "await-send",
+                serde_json::to_string(&send_op)?.substring(1, 65)
```

Comment:
Why do you need to substring the `OperationId`?

Reply:
serde json adds quotes we need to remove